.github/workflows/raydp.yml

```diff
@@ -94,6 +94,7 @@ jobs:
         run: |
           pip install pylint==2.8.3
           pylint --rcfile=python/pylintrc python/raydp
+          pylint --rcfile=python/pylintrc examples/raytrain_nyctaxi.py
```

Comment:
make it examples/*.py?

Reply:
Ok, I'll push again after modifying other files.